builtins.go

```diff
@@ -286,6 +342,30 @@ var builtinAtan = liftNumeric(math.Atan)
 var builtinLog = liftNumeric(math.Log)
 var builtinExp = liftNumeric(math.Exp)
 
+func liftBitwise(f func(int64, int64) int64) func(*evaluator, potentialValue, potentialValue) (value, error) {
+	return func(e *evaluator, xp, yp potentialValue) (value, error) {
+		x, err := e.evaluateNumber(xp)
+		if err != nil {
+			return nil, err
+		}
+		y, err := e.evaluateNumber(yp)
+		if err != nil {
+			return nil, err
+		}
+		xInt := int64(x.value)
+		yInt := int64(y.value)
+		// TODO(sbarzowski) What if the result is inf, nan or otherwise not allowed?
```

Comment:
I don't think it's possible.  You might lose precision though, e.g.

```
$ jsonnet -e '(1 << 62) | 1'
4611686018427387904
```

Reply:
Right. This comment dates back to when I thought float64 binary represenation is taken as is, instead of being converted to an integer.

I've added this example to the test suite.